### ISSUE TITLE
[Bug][Doc]Documents check detected several Missing and Stale in docs

### ISSUE THREAD

**Issue by williamyang1** [flashinfer-ai/flashinfer#3202]
Run document check for release-v0.6.9.
commit: a1aa676196f798435248d9ea205c67674476f473

CC @aleozlx 
Error Log:
`
🔴 Missing from docs 50 functions
flashinfer.decode

    single_decode_with_kv_cache_with_jit_module
    trtllm_batch_decode_with_kv_cache_mla
    xqa_batch_decode_with_kv_cache
    xqa_batch_decode_with_kv_cache_mla
    xqa_mla

flashinfer.fp4_quantization

    block_scale_interleave
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    nvfp4_quantize_paged_kv_cache

flashinfer.fused_moe

    b12x_fused_moe
    fused_topk_deepseek
    trtllm_bf16_moe
    trtllm_bf16_routed_moe
    trtllm_fp4_block_scale_routed_moe
    trtllm_fp8_block_scale_routed_moe
    trtllm_mxint4_block_scale_moe

flashinfer.gemm

    fp8_blockscale_gemm_sm90
    gemm_fp8_nt_blockscaled
    group_gemm_nvfp4_nt_groupwise
    mm_M1_16_K7168_N128
    mm_M1_16_K7168_N256
    mm_fp8
    tgv_gemm_sm100
    tinygemm_bf16

flashinfer.prefill

    fmha_v2_prefill_deepseek
    single_prefill_with_kv_cache_with_jit_module
    trtllm_fmha_v2_prefill
    trtllm_ragged_attention_deepseek

flashinfer.quantization

    block_scale_interleave
    e2m1_and_ufp8sf_scale_to_float
    fp4_quantize
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    mxfp4_quantize_cute_dsl
    mxfp8_dequantize_host
    mxfp8_quantize
    mxfp8_quantize_cute_dsl
    nvfp4_batched_quantize
    nvfp4_quantize
    nvfp4_quantize_cute_dsl
    scaled_fp4_grouped_quantize
    shuffle_matrix_a
    shuffle_matrix_sf_a

flashinfer.rope

    mla_rope_quantize_fp8
    rope_quantize_fp8
    rope_quantize_fp8_append_paged_kv_cache

flashinfer.sampling

    sampling_from_logits
    softmax

🟠 Stale in docs (no @flashinfer_api found) 12 entries
flashinfer

    topk.can_implement_filtered_topk

flashinfer.comm

    create_shared_buffer
    free_shared_buffer
    pack_strided_memory
    vllm_all_reduce
    vllm_dispose
    vllm_get_graph_buffer_ipc_meta
    vllm_init_custom_ar
    vllm_meta_size
    vllm_register_buffer
    vllm_register_graph_buffers

flashinfer.prefill

    single_prefill_with_kv_cache_return_lse

`

**Comment by williamyang1** [comment]
Updated for latest code in main branch
commit:
ed7028336d3fa8fad91b0bb06bbe7cdd39caeb57

##########

🔴 Missing from docs 54 functions
flashinfer.comm

    decode_cp_a2a_allocate_workspace
    decode_cp_a2a_alltoall
    decode_cp_a2a_init_workspace
    decode_cp_a2a_workspace_size

flashinfer.decode

    single_decode_with_kv_cache_with_jit_module
    trtllm_batch_decode_with_kv_cache_mla
    xqa_batch_decode_with_kv_cache
    xqa_batch_decode_with_kv_cache_mla
    xqa_mla

flashinfer.fp4_quantization

    block_scale_interleave
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    nvfp4_quantize_paged_kv_cache

flashinfer.fused_moe

    b12x_fused_moe
    fused_topk_deepseek
    trtllm_bf16_moe
    trtllm_bf16_routed_moe
    trtllm_fp4_block_scale_routed_moe
    trtllm_fp8_block_scale_routed_moe
    trtllm_mxint4_block_scale_moe

flashinfer.gemm

    fp8_blockscale_gemm_sm90
    gemm_fp8_nt_blockscaled
    group_gemm_nvfp4_nt_groupwise
    mm_M1_16_K7168_N128
    mm_M1_16_K7168_N256
    mm_fp8
    tgv_gemm_sm100
    tinygemm_bf16

flashinfer.prefill

    fmha_v2_prefill_deepseek
    single_prefill_with_kv_cache_with_jit_module
    trtllm_fmha_v2_prefill
    trtllm_ragged_attention_deepseek

flashinfer.quantization

    block_scale_interleave
    e2m1_and_ufp8sf_scale_to_float
    fp4_quantize
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    mxfp4_quantize_cute_dsl
    mxfp8_dequantize_host
    mxfp8_quantize
    mxfp8_quantize_cute_dsl
    nvfp4_batched_quantize
    nvfp4_quantize
    nvfp4_quantize_cute_dsl
    scaled_fp4_grouped_quantize
    shuffle_matrix_a
    shuffle_matrix_sf_a

flashinfer.rope

    mla_rope_quantize_fp8
    rope_quantize_fp8
    rope_quantize_fp8_append_paged_kv_cache

flashinfer.sampling

    sampling_from_logits
    softmax

🟠 Stale in docs (no @flashinfer_api found) 12 entries
flashinfer

    topk.can_implement_filtered_topk

flashinfer.comm

    create_shared_buffer
    free_shared_buffer
    pack_strided_memory
    vllm_all_reduce
    vllm_dispose
    vllm_get_graph_buffer_ipc_meta
    vllm_init_custom_ar
    vllm_meta_size
    vllm_register_buffer
    vllm_register_graph_buffers

flashinfer.prefill

    single_prefill_with_kv_cache_return_lse

**Comment by williamyang1** [comment]
Updated the issue with latest 0.6.11


🔴 Missing from docs 57 functions
flashinfer.comm

    decode_cp_a2a_allocate_mnnvl_workspace
    decode_cp_a2a_alltoall
    decode_cp_a2a_init_workspace
    decode_cp_a2a_workspace_size

flashinfer.decode

    single_decode_with_kv_cache_with_jit_module
    trtllm_batch_decode_with_kv_cache_mla
    xqa_batch_decode_with_kv_cache
    xqa_batch_decode_with_kv_cache_mla
    xqa_mla

flashinfer.fp4_quantization

    block_scale_interleave
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    nvfp4_quantize_paged_kv_cache

flashinfer.fused_moe

    b12x_fused_moe
    fused_topk_deepseek
    trtllm_bf16_moe
    trtllm_bf16_routed_moe
    trtllm_fp4_block_scale_routed_moe
    trtllm_fp8_block_scale_routed_moe
    trtllm_mxint4_block_scale_moe

flashinfer.gemm

    fp8_blockscale_gemm_sm90
    gemm_fp8_nt_blockscaled
    group_gemm_nvfp4_nt_groupwise
    mm_M1_16_K7168_N128
    mm_M1_16_K7168_N256
    mm_fp8
    tgv_gemm_sm100
    tinygemm_bf16

flashinfer.norm

    fused_dit_gate_residual_layernorm_gamma_beta
    fused_dit_gate_residual_layernorm_scale_shift
    fused_dit_residual_layernorm_scale_shift

flashinfer.prefill

    fmha_v2_prefill_deepseek
    single_prefill_with_kv_cache_with_jit_module
    trtllm_fmha_v2_prefill
    trtllm_ragged_attention_deepseek

flashinfer.quantization

    block_scale_interleave
    e2m1_and_ufp8sf_scale_to_float
    fp4_quantize
    mxfp4_dequantize
    mxfp4_dequantize_host
    mxfp4_quantize
    mxfp4_quantize_cute_dsl
    mxfp8_dequantize_host
    mxfp8_quantize
    mxfp8_quantize_cute_dsl
    nvfp4_batched_quantize
    nvfp4_quantize
    nvfp4_quantize_cute_dsl
    scaled_fp4_grouped_quantize
    shuffle_matrix_a
    shuffle_matrix_sf_a

flashinfer.rope

    mla_rope_quantize_fp8
    rope_quantize_fp8
    rope_quantize_fp8_append_paged_kv_cache

flashinfer.sampling

    sampling_from_logits
    softmax

🟠 Stale in docs (no @flashinfer_api found) 12 entries
flashinfer

    topk.can_implement_filtered_topk

flashinfer.comm

    create_shared_buffer
    free_shared_buffer
    pack_strided_memory
    vllm_all_reduce
    vllm_dispose
    vllm_get_graph_buffer_ipc_meta
    vllm_init_custom_ar
    vllm_meta_size
    vllm_register_buffer
    vllm_register_graph_buffers

flashinfer.prefill

    single_prefill_with_kv_cache_return_lse

**Comment by williamyang1** [comment]
As added the following new test range for the document check,
1. Docstring Completeness
2. Args Consistency
3. Env Vars Consistency
4. Supported Architecture Consistency
5. CLAUDE.md Quick-Ref Paths Exist 
6. Skill MD References Exist 
7. Docs Index References Exist

More issues found for the latest 0.6.12 code. Please check it. @aleozlx 
And as the data is too big, so please see the result from [flashinfer_0.6.12_doc_check_result](http://dlswqa-nas:8080/cudnn/flashinfer/flashinfer_document_check_report/flashinfer_0.6.12_doc_check_result.html)

**Comment by kangbintNV** [comment]
Most issue has been handled in #3538 and https://github.com/flashinfer-ai/flashinfer/issues/3753, keep tracking doc issue nightly & per each release.

Latest results  [flashinfer_0.6.15rc4_doc_check_result](http://dlswqa-nas:8080/cudnn/flashinfer/flashinfer_document_check_report/flashinfer_0.6.15rc4_0b24eb5b_doc_check_result.html).
